sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/src/TablesExtensionConfigProvider.cs

```diff
@@ -165,6 +189,51 @@ await foreach (var entity in entities)
             return entityArray;
         }
 
+        private async Task<ParameterBindingData[]> CreateEnumerable(TableAttribute attribute, ValueBindingContext context)
```

Comment:
As this will only create Enumerable of ParameterBindingData, we can use the name `CreateParameterBindingDataEnumerable`.